functions.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+
+const functions = {
+    add: (x,y) => {
+        if(typeof x !== 'number' || typeof y !== 'number') {
+            return null;
+        } else {
+            return x + y;
+        }
+    },
+    mult: (num1,num2) => num1 * num2,
+    createUser: () => {
+        const user = { firstName: 'Mike' }
+        user['lastName'] = 'Wood';
+        return user
+    },
+    fetchUser: () => axios.get('https://jsonplaceholder.typicode.com/users/1')
+        .then(res => res.data)
+        .catch((error) => {
+            console.log(error);
```

Comment:
fyi: you removed the test for this code, but that's ok for this test project. In a real one I would not allow that